src/components/frontegg-context/types.ts

```diff
@@ -0,0 +1,27 @@
+import { IRedisCacheOptions } from "../../cache/types";
+
+export interface IFronteggContext {
+    FRONTEGG_CLIENT_ID: string;
+    FRONTEGG_API_KEY: string;
+}
+
+export interface IFronteggOptions {
+    accessTokens?: IAccessTokensOptions;
```

Comment:
maybe rename to accessTokensOptions, it's a bit confusing because accessing options.accessTokens makes it seem list there are accessTokens on the options object

Reply:
done